### PR TITLE
Specify 'any' license in TFA pageimage request

### PR DESCRIPTION
**Pull request by mdholloway**

### CODE REVIEW

File: v1/feed.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ class Feed {
                         prop: 'pageimages',
                         piprop: 'thumbnail',
                         pithumbsize: 640,
+                        pilicense: 'any'
```
Comment (Pchelolo) [inline v1/feed.js]: You forgot a comma

**Comment by mdholloway** [comment]
That's what I get for trying to do a quick PR before standup... 

The test failures look unrelated.

**Comment by Pchelolo** [comment]
Yea the tests are failing because we can't download Cassandra, seem like the mirror on apache.org is down. Let's wait for ti to get back to actually run the tests.

LGTM overall, thank you for a nice catch

**Comment by Pchelolo** [comment]
The tests passed, merging. Thank you @mdholloway

**Pull request merged**